### DISCUSSION TITLE
Use stablemaster collection map for worldwide travel view

### DISCUSSION THREAD

**Discussion by wduda** [wduda/TravelWindowII#229] [Feature Ideas & Requests]
Old feature request for a singular scrollable map with quickslots on it.

**Comment by dt192** [comment]
Will this replace the current maps?

**Reply by wduda** [reply]
They could be made available in parallel.

**Comment by wduda** [comment]
Comment from Discord:
> if you're not replacing these with the new stablemaster map, then a drop down menu up top to switch between the maps would be useful.

![image](https://github.com/user-attachments/assets/8dd326c3-056f-47c1-ad4d-ad1910682bd0)

**Comment by wduda** [comment]
The use of the stablemaster map was rejected - instead a map view was introduced: [v4.0.0 #265](https://github.com/wduda/TravelWindowII/pull/265)
